extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -146,6 +153,16 @@ export class DraggableDrawer extends AMP.BaseElement {
       )
     );
 
+    if (isPageAttachmentUiV2ExperimentOn(this.win)) {
+      const spacerEl = this.win.document.createElement('div');
+      spacerEl.classList.add('i-amphtml-story-draggable-drawer-spacer');
+      this.containerEl_.prepend(spacerEl);
```

Comment:
I'm surprised the linter lets you use this, it used to throw an error and advise to use `insertBefore` that offers the same feature but has better browser support.
Can you switch?